build/items.json

```diff
@@ -5118,29 +5141,16 @@
   },
   "satanic": {
     "hint": [
-      "Active: Unholy RageIncreases Lifesteal percentage to 200% for 6 seconds. Dispel Type: Basic Dispel ",
-      "Passive: LifestealHeals the attacker for 25% of attack damage dealt."
+      "Active: Unholy RageIncreases Lifesteal percentage to %unholy_lifesteal_total_tooltip%%% for %unholy_duration% seconds. Dispel Type: Basic Dispel ",
```

Comment:
It looks like a lot of these replacements are no longer working

Reply:
Yes, that was already the case before, for example here : https://github.com/odota/dotaconstants/blob/master/build/items.json#L50

I started fixing it, but I think there's quite a lot of things to change in order for it be fully working, so I was going to send it in a later PR. Is this blocking for this change ?

Reply:
It appears to be currently working for some but not for others. Either this change or a change in the source data is breaking it for additional cases. It may help to run the updateconstants script without your change separate and update master with that so we can track down the differences

Reply:
Here's a PR for 7.31c without any code changes : https://github.com/odota/dotaconstants/pull/119